src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -516,6 +518,19 @@ public void shouldNotSendContestedConsentOrderApprovedEmail() throws Exception {
         verifyNoInteractions(notificationService);
     }
 
+    @Test
+    public void sendContestedGeneralApplicationReferToJudgeEmail() throws Exception {
```

Comment:
Can we also add a test secnario for a Server Error, similar to GeneralEmail // throwExceptionWhenGeneralEmailIsRequested? 
Just due to the email value being retrieved from case data